quic/s2n-quic-core/src/recovery/bandwidth/estimator.rs

```diff
@@ -26,45 +30,63 @@ pub struct PacketInfo {
     pub is_app_limited: bool,
 }
 
-const MICRO_BITS_PER_BYTE: u64 = 8 * 1000000;
-
-#[derive(Copy, Clone, Debug, Default, Eq, Ord, PartialOrd, PartialEq)]
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
 pub struct Bandwidth {
-    bits_per_second: u64,
+    nanos_per_byte: u64,
```

Comment:
coming back to this later its not going to be obvious why this is time/byte rather than the more obvious byte/time. can you add a comment here inline explaining why we choose to do it this way.